artifactory/services/buildinfo.go

```diff
@@ -0,0 +1,68 @@
+package services
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/jfrog/jfrog-client-go/artifactory/auth"
+	"github.com/jfrog/jfrog-client-go/artifactory/buildinfo"
+	rthttpclient "github.com/jfrog/jfrog-client-go/artifactory/httpclient"
+	"github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+type BuildInfoService struct {
```

Comment:
I think we should merged the existing *PublishBuildInfoService* into this service.

Reply:
Merge the two services into BuildInfoService.